code/find/FindSpaceResorts.js

```diff
@@ -1,5 +1,5 @@
-import findResorts from "./lib/main.js";
+import main from "./lib/main.js";
```

Comment:
I think it should be
```
import * as main
```
no?

Reply:
Yes, you are correct. Works fine that way, thanks for commenting 
![image](https://user-images.githubusercontent.com/102233419/233102191-182177ed-0eb5-41fc-b77d-0854665b482d.png)